frontend/fingertips-frontend/components/linechartsOptions/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { WeatherForecast } from '@/generated-sources/api-client';
+import Highcharts from 'highcharts';
+import { HighchartsReact } from 'highcharts-react-official';
+
+interface LineChartProps {
+  data: WeatherForecast[];
+  title?: string;
+  xAxisTitle?: string;
+  yAxisTitle?: string;
+}
+
+export function LineChart({
+  data,
+  title = 'Weather Forecast',
+  xAxisTitle = 'Date',
+  yAxisTitle = 'Temperature (°C)',
+}: Readonly<LineChartProps>) {
+  const categories = data.map(
+    (item) => item.date?.toLocaleDateString('en-GB') ?? ''
+  );
+  const temperatureData = data.map((item) => item.temperatureC ?? '');
+
+  const lineChartOptions: Highcharts.Options = {
+    chart: { type: 'line' },
+    title: { text: title },
+    xAxis: {
+      categories: categories,
+      title: { text: xAxisTitle },
+    },
+    yAxis: {
+      title: { text: yAxisTitle },
+    },
+    series: [
+      {
+        type: 'line',
+        name: yAxisTitle,
+        data: temperatureData,
+      },
+    ],
+    accessibility: {
```

Comment:
Did you say the accessibility module wasn't working? [This GitHub issue](https://github.com/highcharts/highcharts-react/issues/141) might help to get it sorted. I think we should be trying to include it in this PR, if it doesn't take too much time (and making sure we address it later if not).

Reply:
The fix I put in previously removed the accessibility console error, I removed the fix because when JS was disabled on the browser nothing rendered on the page, no title or table. Im happy to revert it back, but not sure if we want a blank page when JS is disabled.

Reply:
We definitely don't want a blank page. Perhaps not one to worry about in this PR then, as long as we make sure to look at it in a future one.